### DISCUSSION TITLE
FATAL: command failed

### DISCUSSION THREAD

**Discussion by Cassiatora** [NVlabs/instant-ngp#559] [Q&A]

**Comment by thnewlands** [comment]
Hi -- I'm running into this with some datasets I'm trying to create from video with colmap too. Have you had any luck?

**Comment by thnewlands** [comment]
```
==== running: colmap feature_extractor --ImageReader.camera_model OPENCV --SiftExtraction.estimate_affine_shape=true --SiftExtraction.domain_size_pooling=true --ImageReader.single_camera 1 --database_path colmap.db --image_path ".\images"
FATAL: command failed
```

**Comment by thnewlands** [comment]
The issue for me seems to be that colmap_text\\cameras.txt isn't being generated.

**Comment by thnewlands** [comment]
OK!! I got it figured out -- when you add colmap to your path - use colmap's root folder instead of the bin folder. You should be running it from COLMAP.BAT rather than colmap.exe
